shell/src/main/java/org/apache/kafka/shell/MetadataNodeManager.java

```diff
@@ -318,6 +320,16 @@ private void handleCommitImpl(MetadataRecordType type, ApiMessage message)
                     node.create(record.key()).setContents(record.value() + "");
                 break;
             }
+            case PRODUCER_IDS_RECORD: {
+                ProducerIdsRecord record = (ProducerIdsRecord) message;
+                DirectoryNode lastBlockNode = data.root.mkdirs("lastProducerIdBlock");
```

Comment:
I don't feel too strongly, but it seems a little odd to have the next and last blocks at the same level with different fields. How about this instead?

- `/producerIds/nextBlockStartId`
- `/producerIds/lastBlockAssignedBrokerId`
- `/producerIds/lastBlockAssignedBrokerEpoch`

Basically a more literal translation of the `ProducerIdsRecord`.

Reply:
Yeah, better naming, done.